lib/player.js

```diff
@@ -672,6 +672,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     /** @private {?shaka.extern.PlayerConfiguration} */
     this.config_ = this.defaultConfig_();
 
+    /** @private {?number} */
+    this.currentTargetLatency_ = null;
+
+    /** @private {number} */
+    this.rebufferingCount_ = -1;
+
+    /** @private {?number} */
+    this.targetLatencyReached_ = null;
```

Comment:
We should reset this value in the unload method to avoid inconsistencies between loads